src/Stryker.Core/Stryker.Core/StrykerRunner.cs

```diff
@@ -62,7 +62,8 @@ public void RunMutationTest(StrykerOptions options)
                             new AssignmentStatementMutator(),
                             new PrefixUnaryMutator(),
                             new PostfixUnaryMutator(),
-                            new CheckedMutator()
+                            new CheckedMutator(),
+                            new LinqMutator()
```

Comment:
When adding a mutator I expect the documentation to be updated. It should be included in the list of mutators on both the main readme and for the Core project readme.

Reply:
Yep - I've overlooked that part.